base/util/merge_test.go

```diff
@@ -141,9 +142,11 @@ func TestMergeTranslatedConfigs(t *testing.T) {
 		},
 	}
 	for i, test := range tests {
-		c, ts := MergeTranslatedConfigs(test.parent, test.parentTranslations, test.child, test.childTranslations)
-		assert.Equal(t, test.merged, c, "#%d: bad config", i)
-		assert.Equal(t, test.mergedTranslations, ts, "#%d: bad translations", i)
+		t.Run(fmt.Sprintf("merge %d", i), func(t *testing.T) {
+			c, ts := MergeTranslatedConfigs(test.parent, test.parentTranslations, test.child, test.childTranslations)
+			assert.Equal(t, test.merged, c, "#%d: bad config", i)
```

Comment:
If you're up for it, it'd be nice to remove all of the `#%d: ` stuff, since it's now redundant.